spec/controllers/account/news_controller_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.describe Account::NewsController, type: :controller do
+  render_views
+
+  let!(:osbb) { create(:osbb) }
+  let!(:lead) { create(:user, role: :lead, osbb: osbb) }
+  let!(:simple) { create(:user, role: :simple, osbb: osbb) }
+  let!(:news) { create(:news, user: lead, osbb: osbb) }
```

Comment:
When you create news, Factoy bot creates user and osbb. Use: `news.user` or `news.osbb`